src/main/scala/com/typesafe/sbt/packager/docker/DockerSupport.scala

```diff
@@ -2,9 +2,9 @@ package com.typesafe.sbt.packager.docker
 
 object DockerSupport {
 
-  def chownFlag(version: DockerVersion): Boolean =
-    (version.major == 17 && version.minor >= 9) || version.major > 17
+  def chownFlag(version: DockerVersion, apiVersion: DockerApiVersion): Boolean =
+    ((version.major == 17 && version.minor >= 9) || version.major > 17) || (apiVersion.major == 1 && apiVersion.minor >= 32)
 
-  def multiStage(version: DockerVersion): Boolean =
-    (version.major == 17 && version.minor >= 5) || version.major > 17
+  def multiStage(version: DockerVersion, apiVersion: DockerApiVersion): Boolean =
+    ((version.major == 17 && version.minor >= 5) || version.major > 17) || (apiVersion.major == 1 && apiVersion.minor >= 29)
```

Comment:
I think this is a good path for migration. In the end it is probably saner to only check the api version

Reply:
agree

Reply:
I had to restore the previous versions of this methods due to compatibility issues